styles.css

```diff
@@ -2,18 +2,18 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-    max-width: 1440px;
+
 }
 .hero-footer {
     display: flex;
+    width: 100%;
     align-items: center;
-    height: 142px;
     background: #FFFFFF;
+    justify-content: space-around;
```

Comment:
Do these 4 lines affect any styling in the design? check through inspect , if it doesn't, then remove.